tests/chainermn_tests/links_tests/test_multi_node_chain_list.py

```diff
@@ -238,8 +240,9 @@ def check_cycle_model(gpu, param):
     n, d = 100, 10
 
     with chainer.using_config('dtype', param.dtype):
+        io_dtype = chainer.get_dtype(map_mixed16=np.float16)
```

Comment:
What is the intention here on explicitly setting `map_mixed16` ? I'm afraid we don't need this line.

Reply:
Because the mixed16 type is a chainer type, which doesn't exist in numpy. To create a numpy object, mixed16 needs to be maped to a numpy type, which is this line

Reply:
It is fine to manually identify the mixed16 and set to nump.float16. Since chainer has already provided this functionality, it is more chaineric to use chainer.get_dtype

Reply:
mixed16 doesn't need to be mapped if we intend fp16.

```python
>>> import numpy, chainer
>>> numpy.arange(10).reshape(1, 10).astype(chainer.mixed16)
array([[0., 1., 2., 3., 4., 5., 6., 7., 8., 9.]], dtype=float16)
```